platform/platform-configuration-listener/pom.xml

```diff
@@ -118,7 +118,7 @@
                                         <limit>
                                             <counter>BRANCH</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.85</minimum>
```

Comment:
Looks like a ran into the same problem that @ericlessard saw with try-with-resources and Jacoco.  That is why I had to lower the coverage ratio.